index.js

```diff
@@ -135,7 +135,7 @@ threshold'), { code: 'ECOMPROMISED' }));
     // Unref the timer so that the nodejs process can exit freely
     // This is safe because all acquired locks will be automatically released
     // on process exit
-    lock.updateTimeout.unref();
+    if (lock.updateTimeout.unref) lock.updateTimeout.unref();
```

Comment:
Please add brackets to the if body. Also can you live a comment saying that people might be using this module outside node?